pkg/registry/rbac/validation/rule.go

```diff
@@ -361,6 +384,18 @@ func (r *StaticRoles) GetRole(namespace, name string) (*rbacv1.Role, error) {
 	return nil, errors.New("role not found")
 }
 
+func (r *StaticRoles) GetRoleWithMultiTenancy(namespace, tenant, name string) (*rbacv1.Role, error) {
+	if len(namespace) == 0 {
```

Comment:
We also need to check that tenant is not empty.

Reply:
fixed